modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,97 @@
+= Provisioning Fedora CoreOS on Microsoft Hyper-V
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Microsoft Hyper-V.
+
+== Prerequisites
+
+You must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File].
+
+You will also need a small utility from https://github.com/containers/libhvee[libhvee] called `kvpctl`. It attaches your Ignition config to your virtual machine. Precompiled binaries can be found on the project's https://github.com/containers/libhvee/releases[releases page].
+
+=== Downloading the disk image
+
+Hyper-V disk images can be manually downloaded from the https://fedoraproject.org/coreos/download/[download page]. Be sure to decompress the image after downloading.
+
+== Booting a new VM on Microsoft Hyper-V
+
+=== Creating a virtual switch
+
+You must first create a virtual switch so your virtual machine has a network to connect to. To do this, launch Hyper-V Manager and click _Virtual Switch Manager..._ in the _Actions_ panel:
+
+image::hyperv-actions.png[Hyper-V Manager Actions panel]
+
+Then follow the prompts under _New virtual network switch_:
```

Comment:
Note there are a lot of options under this "wizard" I just kept the defaults.. It's interesting to note that I did lose my network connection as part of this (I was warned that it was possible) so my RDP session was terminated.

Not sure if that's worth mentioning here or not.

Reply:
This only happens when configuring an external switch, not an internal or private one, and there is indeed a warning dialog.  Since we say "follow the prompts", I think it's okay to leave it.

Reply:
hmm. should I have configured an internal or private switch instead?

Reply:
Well, it depends what you wanted.  The dialog box explains the effect of each option.

Reply:
got ya. it just wasn't clear to me which one of them would work for out purposes here. if any of them will work maybe we should have a note here that mentions the user can choose any of them based on their preference.

Reply:
i.e. i just went with the default, which is fine, but the default did kick me out of my machine and I had to reconnect. I probably would have chosen `internal` or `private` otherwise.

Reply:
i.e. we could add here some text to the effect of: Any type of switch `External`, `Internal`, or `Private` should suffice.

Reply:
Okay, done.